JS/Lesson24/tugba/filter.js

```diff
@@ -409,3 +424,6 @@ const usersExercise10 = [
 		email: 'sophia@example.com',
 	},
 ];
+const filteredUsers = usersExercise10.filter(user => user.name.startsWith('J'));
```

Comment:
```suggestion
const filteredUsers = usersExercise10.filter(user => user.name.toUpperCase().startsWith('J'));
```
would be a safer choice in case the name is written starting with a lower case letter :)